wafflehaus/edit_response/__init__.py

```diff
@@ -52,27 +52,57 @@ def _replace_lookup(self, replace_str):
             return {}
         return replace_str
 
+    def _foreach(self, conditional, data):
+        if not isinstance(data, list):
+            return data
+        if not all(isinstance(item, dict) for item in data):
```

Comment:
I don't think mixed types makes sense since it's checking on attributes. I don't think going down N-levels is useful in this situation.